File: CSHARP/CSHARP/Model/Igrac.cs
using System;
namespace CSHARP.Model
{
    public class Igrac: Entitet
    {
        public string ime { get; set; }
        public string prezime { get; set; }
        public string urlSlika { get; set; }
        public int spol { get; set; }

        public override string ToString()
        {
            return ime + " " + prezime;
        }
    }
}

File: Kotlin/src/main/kotlin/model/Igrac.kt
package model

/**
 * @author Matija Skrletović
 **/

class Igrac(sifra: Int, val ime: String, val prezime: String, val urlSlike: String, val spol: Int) : Entitet(sifra) {

    override fun toString() = "$ime $prezime"

}